string region_id
uint32 num_candidates
std_msgs/Float32MultiArray points 
